### DISCUSSION TITLE
接口可以支持gpu的高效率batch推理吗

### DISCUSSION THREAD

**Discussion by cmathx** [PaddlePaddle/PaddleOCR#13172] [Q&A]
每次只处理一张图片，性能很低，在GPU场景下是否支持batch推理，即一次请求同时处理多张图片？

**Comment by GreatV** [comment]
@cmathx 你愿意提交一个PR来支持这个功能吗

**Comment by cmathx** [comment]
我手写了一版，没有完全达到并发的效果。
看到paddle官方这边提供了多线程支持
https://github.com/PaddlePaddle/FastDeploy/blob/develop/tutorials/multi_thread/python/pipeline/multi_thread_process_ocr.py

**Comment by cmathx** [comment]
这里有一个问题。cpu启动了多线程，top -H监控查看确实是ok的。怀疑提交的predict_image，gpu没有做并行。
